stubs/webapp2/__init__.pyi

```diff
@@ -55,3 +72,33 @@ class WSGIApplication:
             environ: Dict[str, str],
             start_response: Response
     ) -> Response: ...
+
+class Route:
+    def __init__(
+            self,
+            template: Union[str, Pattern[Any]],
```

Comment:
I have added stubs for webapp2 by reading the doc itself(https://webapp2.readthedocs.io/en/latest/api/webapp2.html), because there is no pre-defined stubs in typesheds and there is not much typing info in the source code itself. So, that's why there are `Any`s -- but i have tried my best to reduce the Any. PTAL!